test/type/bignumber/function/bignumber.test.js

```diff
@@ -80,6 +80,17 @@ describe('bignumber', function () {
     assert.strictEqual(a.toString(), '0.33333333333333333333333333333333')
   })
 
+  it('should support very high precisions', function () {
```

Comment:
This new test passed even before the fix. The problem was with `math.format`, not `toString`.